server/models/AssessmentDayDataModel/index.js

```diff
@@ -21,10 +21,6 @@ const AssessmentDayDataModel = {
     await db
       .collection(collections.assessmentDayData)
       .insertMany(assessmentDayData),
-  update: async (db, query, assessmentDayDataAttributes) =>
-    await db
-      .collection(collections.assessmentDayData)
-      .updateOne(query, { $set: assessmentDayDataAttributes }),
```

Comment:
just checking - was think function referenced anywhere else in the codebase?

Reply:
nope I scanned for other occurrences and there is just this one.